### PR TITLE
Feature/customisations - custom icon background

### PR DESCRIPTION
**Pull request by kopijunkie**
# Feature
We want to offer customers supported customisation options for the JS SDK so that they can customise the SDK pictogram icons' background colour, i.e. the grey background circle. 

This PR also addresses an inconsistency with the background colour of the icons in the Document Type Selector options where Resident Permit icon's background colour is a slightly darker shade of grey than the rest.

SDK pictogram icons have also been updated to not overflow the circle background, e.g. some of the icons on Cross Device Intro screen.
SDK Document Type Selector screen - default themeSDK Document Type Selector screen - customised theme
SDK Document Upload screen - default themeSDK Document Upload screen - customised theme
SDK Cross Device Intro screen - default themeSDK Cross Device Intro screen - customised theme

Out of scope: Support for custom icon images.

## Checklist

_put `n/a` if item is not relevant to PR changes_

- [x] Has the CHANGELOG been updated?
- [x] Has the README been updated?
- [ ] Has the CONTRIBUTING doc been updated?
- [ ] Has the RELEASE_GUIDELINES been updated?
- [ ] Has the MIGRATION doc been updated for any MAJOR breaking changes?
- [ ] Has the MIGRATION doc been updated for any MINOR breaking changes, including any translation strings or keys changes?
- [ ] Have any new automated tests been implemented or the existing ones changed?
- [ ] Have any new manual tests been written down or the existing ones changed?
- [ ] Have any new strings been translated or the existing ones changed?

### CODE REVIEW

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1348-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1348-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://1348-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1348-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://1348-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1348-pr-onfido-sdk-ui-onfido.surge.sh

**Review by ethan605 — APPROVED** [review]
LGTM!

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1348-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://1348-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1348-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1348-pr-onfido-sdk-ui-onfido.surge.sh

File: src/components/DocumentSelector/assets/icon-bank-building-society-statement.svg [hunk 1]
```diff
@@ -1,23 +1 @@
...
...A4eJx1kc1q20AUhc8ktkmgMaWblpLFdFuskeTu7FWiGBMIzsIly8AgD7JAaIQkcO19HqMPkFWX3WeR5+oZeWwIJRru8N1zzxz9AfiIVwjsrwlrzwLn7PZ8gh4Sz6f4jLnnHoZ49NzHBxSeB7jAL89DfMcTT4neObsafzwLfBLfPJ/gTPzwfAolpp57uBS55z6+iEPOAF/Fb89D/BR/b/Isb/OdWcmVbrVMbbWt82zdypdnOY7iKOAWj+Tc2qwwMrF1ZWvd5rZU95Up5VKXzbUtVrGKo2l8tUymTnZq4OSjR7ruwdQNT0pnfmNbt201CcPNZqN0pdO1UbbOwiJPTdmYJry7TWaL5SwYqwg3yJGxWtYOBitIlmavSSksKmz5sZxrTVXimTVGhJgVeIoxojqn29JXMEfyN1meq7pdd/kWJRTuqRmSxJJ6iQbXnBS8a8ypS51yv+I0IR3cB29wdP+fI4+zB05qKvt7ymPy+2nu3VpOJwi5Nt1SdFWslFPDzr1JxmnB3LTLabg3VO5wy6edYcHkWfdVFKJ/n62AnnicY2BmAIP/aQxpDJiAFQAp8gHU) format("woff"); font-weight:bold;font-style:normal;}</style></defs><style><![CDATA[.B{stroke:#2c3e4f}.C{stroke-width:2}.D{fill:#2c3e4f}.E{font-family:OpenSans-Bold, Open Sans}.F{font-weight:bold}]]></style><g fill-rule="evenodd"><circle cx="24" cy="33" r="8" fill="#fff" class="B C"/><text font-size="11" class="D E F"><tspan x="20" y="37">€</tspan></text><circle cx="33" cy="21" r="8" fill="#fff" class="B C"/><text font-size="11" class="D E F"><tspan x="30" y="25">£</tspan></text><circle cx="17" cy="17" r="8" fill="#fff" class="B C"/><text font-size="11" class="D E F"><tspan x="14" y="21">$</tspan></text></g></svg>
```
Comment (stefaniacardenas) [inline src/components/DocumentSelector/assets/icon-bank-building-society-statement.svg:1]: It would be good to pretty print all the SVGs.

**Review by stefaniacardenas — APPROVED** [review]
LGTM, just a minor comment:

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1348-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1348-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1348-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1348-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1348-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://1348-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1348-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1348-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1348-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://1348-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1348-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1348-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://1348-pr-onfido-sdk-ui-onfido.surge.sh

**Pull request merged**